localization/languages/ar.json

```diff
@@ -5,186 +5,186 @@
         "settingsPrivacyHeading": "منع المحتوى",
-        "settingsContentBlockingLevel0": "السماح لجميع الاعلانات و المتعقبين",
-        "settingsContentBlockingLevel1": "حظر اعلانات الطرف التالت و المتعقبين",
-        "settingsContentBlockingLevel2": "حظر جميع الاعلانات و المتعقبين",
-        "settingsContentBlockingExceptions": "سيظل مسموحًا بعرض الاعلانات على هده المواقع",
-        "settingsBlockScriptsToggle": "منع السكريبت",
+        "settingsContentBlockingLevel0": "السماح بإظهار جميع الإعلانات والتعقب",
+        "settingsContentBlockingLevel1": "منع إعلانات الأطراف الثالثة والتعقب",
+        "settingsContentBlockingLevel2": "منع جميع الإعلانات والتعقب",
+        "settingsContentBlockingExceptions": "السمحا بعرض الإعلانات في هذه المواقع",
+        "settingsBlockScriptsToggle": "منع جافاسكربت",
         "settingsBlockImagesToggle": "منع الصور",
         "settingsEasyListCredit": {
-             "unsafeHTML": " استنادا الى EasyList و EasyPrivacy, التي تم انشائها من طرف <a target=\"_blank\" href=\"https://easylist.to/\"> EasyList المؤلفون ل</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">مشاهدة الترخيص</a>)." 
+             "unsafeHTML": " باستعمال قائمتي «إيزي لست» و«إيزي برايفاسي» <a target=\"_blank\" href=\"https://easylist.to/\">لأصحاب «إيزي ليست»</a> (<a target=\"_blank\" href=\"https://creativecommons.org/licenses/by-sa/3.0/legalcode\">قراءة الرخصة</a>)." 
             },
-        "settingsAppearanceHeading": "تفضيلات",
-        "settingsDarkModeToggle": "تشغيل الوضع الليلي",
-        "settingsHistoryButtonToggle": "تمكين زر العودة",
-        "settingsAdditionalFeaturesHeading": "مميزات اضافية",
-        "settingsUserscriptsToggle": "تمكين سكريبت المستخدم",
-        "settingsUserscriptsExplanation": {"unsafeHTML": "يمكن سكريبت المستخدم من  تغير سلوك المواقع - <a href=\"https://github.com/minbrowser/min/wiki/userscripts\">معرفة المزيد</a>."},
-        "settingsUpdateNotificationsToggle": null, //missing translation
-        "settingsSearchEngineHeading": "محرك بحث",
-        "settingsDefaultSearchEngine": "اختر محرك بحث اساسي",
-        "settingsDDGExplanation": "كمحرك بحث اساسي لرؤية النتائج الآنية في مربع البحث DuckDuckGo ",
-        "customSearchEngineDescription": "%s غير كلمة البحث ب",
-        "settingsKeyboardShortcutsHeading": "اختصار لوحة مفاتيح",
-        "settingsKeyboardShortcutsHelp": "اسعمل الفاصلة للفصل بين الاختصارات",
+        "settingsAppearanceHeading": "التفضيلات",
+        "settingsDarkModeToggle": "تشغيل الوضع الداكن",
+        "settingsHistoryButtonToggle": "إظهار زر الرجوع",
+        "settingsAdditionalFeaturesHeading": "المميزات المضافة",
+        "settingsUserscriptsToggle": "تشغيل جافاسكربت المستخدم",
+        "settingsUserscriptsExplanation": {"«إتش تي إم إل» غير آمن": "اكتب نص جافاسكربت لتغير كيفية تصفح المواقع - <a href=\"https://github.com/minbrowser/min/wiki/userscripts\">اقرأ المزيد</a>."},
```

Comment:
The "unsafeHTML" label here isn't part of the translation, so it should remain in English.